src/typography/Code/Code.tsx

```diff
@@ -1,7 +1,7 @@
 import { merge } from "@utilities/merge";
 import React, { FC, PropsWithChildren } from "react";
 
-type CodeHeight = "medium" | "strong";
+type CodeHeight = "default" | "strong" | /* DEPRECATE */ "medium";
```

Comment:
When using TypeScript this will fail on most setups anyway. I think we can change it instead of deprecate it and make it a breaking change as we are in an early stage anyways 🤔

Reply:
@jonasbuechel That's up to you, but I'd really like to bundle breaking changes together into bigger releases when we can avoid it.

Reply:
Well, bundling lot of breaking changes means usually it's a bigger hurdle to bring it to Clarify. I think we should aim for smaller releases in general.